src/serializers/serializeStaticFilters.ts

```diff
@@ -1,6 +1,7 @@
+import { NumberRangeValue } from '../models/searchservice/common/NumberRangeValue';
 import { CombinedFilter } from '../models/searchservice/request/CombinedFilter';
 import { Filter } from '../models/searchservice/request/Filter';
-import { StaticFilters } from '../models/searchservice/request/StaticFilters';
+import { StaticFilters, Filter as InternalFilter } from '../models/searchservice/request/StaticFilters';
```

Comment:
Thoughts on `ApiFilter`? We could even rename it in the StaticFilters file since it's not part of our public API. Having `ApiStaticFilters` could be helpful too.